src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/CpsBulkFileRestoreCoordinationEvent.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Common;
+
+namespace NuGet.VisualStudio
+{
+    public class CpsBulkFileRestoreCoordinationEvent : TelemetryEvent
+    {
+        private const string CpsBulkFileRestoreCoordinationEventName = "CpsBulkFileRestoreCoordination";
+
+        public CpsBulkFileRestoreCoordinationEvent() : base(CpsBulkFileRestoreCoordinationEventName)
```

Comment:
So there will be no property in this newly created event, right?
Just out of curiosity, may I know how will we measure with only the event?

Reply:
@jebriede has more info on that. My understanding no property was required for this telemetry event. We can chat offline how this can be measured. Hope you are okay with that.

Reply:
This purpose of the event is to serve as a marker for creating triggered scorecard for the A/B experiment.

Reply:
Good to know. Thanks.